src/adzerk/boot_cljs/impl.clj

```diff
@@ -35,14 +36,16 @@
        reverse
        (map #(.getPath (target-file-for-cljs-ns % (:output-dir opts))))))
 
-(defn handle-ex [e dirs]
+(defn handle-ex [e dirs report-atom]
   (let [{:keys [type] :as ex} (ex-data (.getCause e))]
     (cond
       (= :reader-exception type)
       (let [{:keys [file line column]} ex
             msg  (some-> e (.getCause) (.getMessage))
-            path (util/find-relative-path dirs file) ]
-        (fail "ERROR: %s on file %s, line %d, column %d\n" msg path line column))
+            path (util/find-relative-path dirs file)
+            exs  (format "ERROR: %s on file %s, line %d, column %d\n" msg path line column)]
+        (swap! report-atom assoc :exception exs)
+        (butil/fail exs))
 
       :default (throw e))))
```

Comment:
Other exceptions could also be interesting.

Figwheel just displays the stacktrace for exceptions which it doesn't have special handling for.